src/main/java/ai/deepcode/javaclient/core/AnalysisDataBase.java

```diff
@@ -351,10 +351,17 @@ private boolean isNotSucceed(
           progress, PREPARE_FILES_TEXT + fileCounter + " of " + totalFiles + " files done.");
 
       final String path = pdUtils.getDeepCodedFilePath(file);
+
+      // don't analyse empty files
+      if (pdUtils.getFileSize(file) == 0) {
```

Comment:
Seems better place for that check will be `DeepCodeUtilsBase#isSupportedFileFormat`
Then we'll have correct files at `getAllSupportedFilesInProject` too.